sechub-pds/src/main/resources/application-pds_integrationtest.yml

```diff
@@ -29,10 +29,10 @@ sechub:
                           # time
    techuser:
        userid: pds-inttest-techuser
-       apitoken: "{noop}pds-inttest-apitoken"
```

Comment:
This is fine, so we test the new "add noop when missing" mechanism automatically when integration tests are executed by GitHub actions.